### DISCUSSION TITLE
Group sample code

### DISCUSSION THREAD

**Discussion by evildrome** [kevinzg/facebook-scraper#640] [Q&A]
Hi All,

 Is there a bit of example code for group scraping?
 
 Even just a one line example of how the parameters are set  (preferably for a public group I could actually test with).
 
 Cheers!
 
 Wilson.

**Comment by neon-ninja** [comment]
Sure:

```python
from facebook_scraper import *
from pprint import pprint

enable_logging()
#set_cookies("cookies.json")

for post in get_posts(group="110354088989367"):
  pprint(post)
```

**Reply by evildrome** [reply]
Thanks Nick!

I had my own C# program written by a very nice Russian guy who had to stop working for me due to mental health issues.

It died after FB made some sort of format change.

I have been DL'ing groups for people for the last 20 years. First Yahoo & then phpBB and later FB.

I kind of gave up after the C# program died but I was contacted by a young woman whose baby had Infant Leukaemia and she made a group for him.  He has recovered (yah!) and she wants to DL the group.

I get a depressing number of requests of this type ( the majority of which do not have a happy ending I can tell you).  

I felt I had to try and help this lady.

Thanks again!

Wilson.

www.personalgroupware.com
